routes/users.js

```diff
@@ -855,7 +856,10 @@ router.post("/register", async (req, res) => {
             _id: userSaved._id,
             name: userSaved.name,
             email: userSaved.email,
-            location: userSaved.location
+            location: userSaved.location,
+            likes:userSaved.likes,
+            dislikes:userSaved.dislikes,
+            factors:userSaved.factors
```

Comment:
As I said - we don't need to repeat that a lot of times

Reply:
May be while fixing error removed item was updated. Fixed it

Reply:
fixed it